### PR TITLE
Added solution for exercise ci_2.04

### PR DESCRIPTION
**Pull request by Nabegon**
Pull request for exercise ci_2.04

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+build.gradle
```
Comment (paraqles) [inline .gitignore]: I would not include this in the `.gitignore` => results in the situation, that the build description file is ignored from git.

File: Greets.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+public class Greets{
+	public static void main (String[] args){
+	System.out.println ("Whats your name :D ?: ");
+	String input = new java.util.Scanner(System.in) .nextLine(); //With nextLine(), we can include blank	
+	System.out.println("Hey! "+ input);
+
```
Comment (paraqles) [inline Greets.java]: A little bit more indentation to make it more obvious that the code is inside the `main` method. But mostly only important if the method is long.

File: Greetstime.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	if(timeinhours >= 0 && timeinhours < 12){
+		System.out.println("Good morning, "+ input + ". Now is "+ timeinhours);
+	}else if (timeinhours >= 12 && timeinhours < 16){
+		System.out.println("Good afternoon, "+ input + ". Now is "+timeinhours);
+	}else if (timeinhours >= 16 && timeinhours < 21){
+		System.out.println("Good evening, " + input + ". Now is "+ timeinhours);
+	}else if (timeinhours >= 21){
+		System.out.println("Good night, "+input+"now is "+timeinhours);
+		}
+	}
```
Comment (paraqles) [inline Greetstime.java]: Best practice => consider including an `else` here

File: Reverse.java [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+		String reverse = "";
+		
+		for(int i = input.length() -1; i >= 0; i--){
+			reverse = reverse + input.charAt(i);
+		}
+		System.out.println("Reversed string is: " + reverse);
+	}
+} 
+
+String a="abc";
```
Comment (paraqles) [inline Reverse.java]: Beginning from here: What is this about?

File: .gitignore (hunk 1 above)
Comment (skoenen) [inline .gitignore]: I would not include this in the `.gitignore` => results in the situation, that the build description file is ignored from git.

File: Greets.java [hunk 5]
```diff
@@ -0,0 +1,8 @@
+public class Greets{
+	public static void main (String[] args){
```
Comment (skoenen) [inline Greets.java]: A little bit more indentation to make it more obvious that the code is inside the `main` method. But mostly only important if the method is long.

File: Greetstime.java [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+	
+	if(timeinhours >= 0 && timeinhours < 12){
+		System.out.println("Good morning, "+ input + ". Now is "+ timeinhours);
+	}else if (timeinhours >= 12 && timeinhours < 16){
+		System.out.println("Good afternoon, "+ input + ". Now is "+timeinhours);
+	}else if (timeinhours >= 16 && timeinhours < 21){
+		System.out.println("Good evening, " + input + ". Now is "+ timeinhours);
+	}else if (timeinhours >= 21){
+		System.out.println("Good night, "+input+"now is "+timeinhours);
+		}
```
Comment (skoenen) [inline Greetstime.java]: Best practice => consider including an `else` here

File: Reverse.java (hunk 4 above)
Comment (skoenen) [inline Reverse.java]: Beginning from here: What is this about?

**Pull request merged**